pandas/tests/arrays/test_datetimelike.py

```diff
@@ -49,9 +50,7 @@ def period_index(freqstr):
     # TODO: non-monotone indexes; NaTs, different start dates
     with warnings.catch_warnings():
         # suppress deprecation of Period[B]
-        warnings.filterwarnings(
-            "ignore", message="Period with BDay freq", category=FutureWarning
-        )
```

Comment:
this can be removed independently of this PR, right?

do you want to open a separate PR to remove this?